packages/samples/react/src/components/handout/basic.tsx

```diff
@@ -48,18 +48,18 @@ const TABLE_HEADERS: KoliBriTableHeaders = {
 				asTd: true,
 			},
 			{
-				label: 'Werktage',
+				label: 'Workdays',
 				colSpan: 5,
 			},
 			{
-				label: 'Wochenende',
+				label: 'Weekend',
 				colSpan: 2,
 			},
 		],
 		[
 			{
-				key: 'montag',
-				label: 'Montag',
+				key: 'monday',
```

Comment:
@anicyne Achtung. Wenn Du den Key änderst, dann musst du auch den Key in den Data anpassen.